headless-services/spring-boot-language-server/src/main/java/org/springframework/ide/vscode/boot/java/livehover/v2/SpringProcessCommandHandler.java

```diff
@@ -301,11 +301,11 @@ private CompletableFuture<Object> handleLiveMetricsProcessRequest(ExecuteCommand
 		String metricName = getArgumentByKey(params, "metricName");
 		if (processKey != null) {
 			switch(metricName) {
-				case "gcPauses": {
+				case SpringProcessConnectorService.GC_PAUSES: {
 					SpringProcessGcPausesMetricsLiveData data = connectorService.getGcPausesMetricsLiveData(processKey);
 					return CompletableFuture.completedFuture(data.getGcPausesMetrics());
 				}
-				case "memory": {
+				case SpringProcessConnectorService.MEMORY: {
```

Comment:
The `metricName` passed here are ("gcPauses", "heapMemory", "nonHeapMemory"). This is used to fetch the region-specific metric data separately. 
(This is the result of my bad naming convention again ;-) )

@BoykoAlex I would like to discuss and get your feedback on how to improve this.